plugins/kn-admin/cmd/private-registry/enable.go

```diff
@@ -0,0 +1,125 @@
+/*
+Copyright © 2020 NAME HERE <EMAIL ADDRESS>
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package private_registry
+
+import (
+	"fmt"
+	"github.com/knative/client-contrib/plugins/kn-admin/cmd"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"os"
+
+	"encoding/json"
+	"github.com/spf13/cobra"
+)
+
+type prcmdFlags struct {
+	DockerServer   string
+	SecretName     string
+	DockerEmail    string
+	DockerUsername string
+	DockerPassword string
+}
+
+type DockerRegistry struct {
+	Auths Auths `json:"auths"`
+}
+type UsIcrIo struct {
+	Username string `json:"Username"`
+	Password string `json:"Password"`
+	Email    string `json:"Email"`
+}
+type Auths struct {
+	UsIcrIo UsIcrIo `json:"us.icr.io"`
+}
+
+var prflags prcmdFlags
+
+// enableCmd represents the enable command
+
+func NewPrEnableCommand(p *cmd.AdminParams) *cobra.Command {
+	var prEnableCmd = &cobra.Command{
+		Use:   "enable",
+		Short: "A brief description of your command",
+		Long: `A longer description that spans multiple lines and likely contains examples
+and usage of using your command. For example:
+
+Cobra is a CLI library for Go that empowers applications.
+This application is a tool to generate the needed files
+to quickly create a Cobra application.`,
+		Run: func(cmd *cobra.Command, args []string) {
+			dockerCfg := DockerRegistry{
+				Auths: Auths{
+					UsIcrIo: UsIcrIo{
+						Username: prflags.DockerUsername,
+						Password: prflags.DockerPassword,
+						Email:    prflags.DockerEmail,
+					},
+				},
+			}
+
+			j, err := json.Marshal(dockerCfg)
+			if err != nil {
+				panic(err)
```

Comment:
We should consistently handle errors that result in panic by calling a common function. Also ensure to give user some message. I would also rather os.Exit(1) rather than just panic.